build/build-compcert.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+## $1 : version (master/x.xx)
+## $2 : Arch
+## $3 : destination: a directory or S3 path (eg. s3://...)
+## $4 : Last revision successfully build (optional)
+
+set -ex
+
+ROOT=$PWD
+VERSION="${1}"
+ARCH="${2}"
+LAST_REVISION="${3-}"
```

Comment:
```suggestion
LAST_REVISION="${4-}"
```

Reply:
Indeed, I changed it

Reply:
Oh, I didn't know you could change directly in my fork.

Reply:
Oh yes sorry, should have make that clear. For easy fix like that, easier to commit it along the original change. Hope it's ok with you?

Reply:
I'm okay. It's a silly mistake